lumimqtt.json.sample

```diff
@@ -1,5 +1,6 @@
 {
     "mqtt_host": "localhost",
+    "client_id": "",
```

Comment:
`client_id` cannot be empty. I think it shouldn't be in the sample file and the user must add it manually if he wants to.